### PR TITLE
Config grunt

### PR DESCRIPTION
**Pull request by crookedneighbor**
Moved the config.json file back to the root of the directory.

Changed postinstall task to use the default grunt task, which is `build:prod` in production and `build:dev` in any other environment.

### CODE REVIEW

File: website/src/utils.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var nodemailer = require('nodemailer');
 var nconf = require('nconf');
+nconf.file({ file: './config.json' });
```
Comment (crookedneighbor) [inline website/src/utils.js]: Thought this path would be relative to `website/src/utils.js`, but it's actually relative to the root of the directory.

File: Gruntfile.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*global module:false*/
 var _ = require('lodash');
+var config = require('./config');
```
Comment (paglias) [inline Gruntfile.js]: heroku doesn't use config.json but passes variables through the environment so i would check for `process.env.NODE_ENV == 'production'`

File: website/src/utils.js (hunk 1 above)
Comment (paglias) [inline website/src/utils.js]: why this? we already load the file here https://github.com/crookedneighbor/habitrpg/blob/config-grunt/website/src/utils.js#L120

Comment (crookedneighbor) [inline website/src/utils.js]: Ahh, I just missed it. Thanks.

Comment (paglias) [inline website/src/utils.js]: @crookedneighbor give a look to the line I linked above, it uses __dirname which i think refers to the path from which the script is run so the root of the repository I think but I'm not sure

File: website/src/utils.js [hunk 3]
```diff
@@ -116,7 +116,7 @@ module.exports.setupConfig = function(){
   nconf.argv()
     .env()
     //.file('defaults', path.join(path.resolve(__dirname, '../config.json.example')))
-    .file('user', path.join(path.resolve(__dirname, '../config.json')));
+    .file('user', path.join(path.resolve(__dirname, './../../config.json')));
```
Comment (paglias) [inline website/src/utils.js:119]: if the path load the file correctly ( I can't test it) then feel free to merge!

**Pull request merged**